packages/kit-bg/src/vaults/impls/evm/sdkEvm/EvmApiProvider.ts

```
@@ -462,6 +475,52 @@ class EvmApiProvider extends BaseApiProvider {
 
+  protected async fallbackEstimateMaxPriorityFeePerGas(): Promise<string> {
+    const BLOCK_COUNT = 3;
+    const PRIORITY_FEE_RATIO = 0.25; // use 25% of base fee as priority fee
+
+    try {
+      // get latest block
+      const latestBlock = await this.client.call<any>('eth_getBlockByNumber', [
+        'latest',
+        false,
+      ]);
+      const latestNumber = parseInt(latestBlock.number, 16);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)